nxt/locator.py

```diff
@@ -150,7 +151,6 @@ def find_one_brick(host=None, name=None, silent=False, strict=None, debug=False,
     try the 'nxt_test' script located in /bin or ~/.local/bin""")
     raise BrickNotFoundError
 
-
```

Comment:
Are you going for PEP8 here? If so, you shouldn't remove these lines (this one and one more below) since they are top level functions.

"Surround top-level function and class definitions with two blank lines."

https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Thanks for the hint, I didn't know PEP8. I tried to make blank lines consistent. In this file there were more 1-blank lines than 2-blank lines to separate top level functions.

Reply:
In 32c84aa99c7f11d0ad196a41c38127c740dc0217 I added 2 blank lines.